generator/config/query/all.yaml

```diff
@@ -11,4 +11,34 @@ arguments:
         name: value
         type:
             - any
+            - fieldQuery
```

Comment:
Is it redundant to specify `any` alongside `fieldQuery`? Looking at `generator/config/expressions.php`, I see the case for `fieldQuery` utilizes the following:

```
'acceptedTypes' => [Type\FieldQueryInterface::class, ...$bsonTypes['any']],
```

Reply:
Indeed, reading the doc I hadn't identified that scalars were used as the implicit `$eq` here. Fixed.